### PR TITLE
Update powershell-preview CHANGELOG for v2021.2.0-preview

### PR DESCRIPTION
**Pull request by andschwa**
Updates the changelog and bumps the version for one more preview. Part of #3178

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
 # PowerShell Extension Release History
 
+## v2021.2.0-preview
+### Friday, February 19, 2021
+#### [vscode-PowerShell](https://github.com/PowerShell/vscode-PowerShell)
+
+- 🛫 🐛 [vscode-powershell #3132](https://github.com/PowerShell/vscode-powershell/pull/3132) -
+  Fix for issue #3131 in regards to brew discarding cask as an option. (Thanks @philippkahr!)
+- 📺 🐛 [vscode-powershell #3177](https://github.com/PowerShell/vscode-powershell/pull/3177) -
+  Delete VS Code Notebook API prototype.
```
Comment (rjmholt) [inline CHANGELOG.md:10]:
```suggestion
  Disable VSCode Notebook API prototype while the API is still unstable.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+- 👷 ✨ [vscode-powershell #3162](https://github.com/PowerShell/vscode-powershell/pull/3162) -
+  Rewrite release signing pipeline #3162.
```
Comment (rjmholt) [inline CHANGELOG.md:12]:
```suggestion
  Rewrite release signing pipeline.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
 # PowerShell Extension Release History
 
+## v2021.2.0-preview
+### Friday, February 19, 2021
+#### [vscode-PowerShell](https://github.com/PowerShell/vscode-PowerShell)
+
+- 🛫 🐛 [vscode-powershell #3132](https://github.com/PowerShell/vscode-powershell/pull/3132) -
+  Fix for issue #3131 in regards to brew discarding cask as an option. (Thanks @philippkahr!)
```
Comment (rjmholt) [inline CHANGELOG.md:8]:
```suggestion
  Use new brew cask installation syntax for PowerShell installation. (Thanks @philippkahr!)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+- 📺 🐛 [vscode-powershell #3177](https://github.com/PowerShell/vscode-powershell/pull/3177) -
+  Delete VS Code Notebook API prototype.
+- 👷 ✨ [vscode-powershell #3162](https://github.com/PowerShell/vscode-powershell/pull/3162) -
+  Rewrite release signing pipeline #3162.
+- 💎 🐛 [vscode-powershell #3129](https://github.com/PowerShell/vscode-powershell/pull/3129) -
+  Add braces to PSCustomObject snippet to play nice with auto-closing pairs.
+- 📖 ✨ [vscode-powershell #3152](https://github.com/PowerShell/vscode-powershell/pull/3152) -
+  Add self as maintainer to readme.
```
Comment (rjmholt) [inline CHANGELOG.md:16]:
```suggestion
  Add @andschwa to the README as maintainer.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,55 @@
...
+- 👷 🐛 [vscode-powershell #3099](https://github.com/PowerShell/vscode-powershell/pull/3099) -
+  Update dependencies to fix CI on macOS.
+- 🐢 🐛 [vscode-powershell #3089](https://github.com/PowerShell/vscode-powershell/pull/3089) -
+  Remove hardcoded output format.
+- 📖 🐛 [vscode-powershell #3086](https://github.com/PowerShell/vscode-powershell/pull/3086) -
+  Fix typo in description of legacy codelens description. (Thanks @nexxai!)
+- 🚂 ✨ [vscode-powershell #3053](https://github.com/PowerShell/vscode-powershell/pull/3053) -
+  Send telemetry from pses.
```
Comment (rjmholt) [inline CHANGELOG.md:24]:
```suggestion
  Use VSCode telemetry endpoint for PSES telemetry.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,55 @@
...
+- 🚂 ✨ [vscode-powershell #3047](https://github.com/PowerShell/vscode-powershell/pull/3047) -
+  Update language server client.
+- 🐢 ✨ [vscode-powershell #2998](https://github.com/PowerShell/vscode-powershell/pull/2998) -
+  Added Pester v5 support to problem matcher. (Thanks @fflaten!)
+- ✂️ 🐛 [vscode-powershell #3023](https://github.com/PowerShell/vscode-powershell/pull/3023) -
+  Help completion should only run on powershell scripts.
```
Comment (rjmholt) [inline CHANGELOG.md:30]:
```suggestion
  Ensure help completions only run on powershell scripts.
```

**Review by rjmholt — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,55 @@
 # PowerShell Extension Release History
```
Comment (SydneyhSmith) [inline CHANGELOG.md:2]: v2021.2.0
Monday, February 22, 2021
📺✨ New semantic highlighting. (Thanks @justinytchen!)
🐢 ✨ Added Pester v5 support to problem matcher. (Thanks @fflaten!)
✨👮 Updated PSScriptAnalyzer to 1.19.1.
- Fixes formatting bugs! (Thanks @bergmeister!)
🐛 Many squashed bugs

Comment (andschwa) [reply CHANGELOG.md:2]: Thank you!

**Pull request merged**